org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -21,7 +21,7 @@ GithubOidcSageBionetworks:
 GithubOidcSageBionetworksIt:
   Type: update-stacks
   DependsOn: GithubOidcSageBionetworks
-  Template: github-oidc-provider-access.j2
+  Template: github-oidc-provider-access.njk
```

Comment:
org-formation uses the nunjucks not jinja template engine that's why we are changing the file extension.